src/components/templates/Audience/Audience.tsx

```diff
@@ -150,7 +150,7 @@ export const Audience: React.FC<AudienceProps> = ({ venue }) => {
   useLayoutEffect(() => {
     if (!venue) return;
 
-    setIframeUrl(ConvertToEmbeddableUrl(venue.iframeUrl, true));
+    setIframeUrl(ConvertToEmbeddableUrl(venue.iframeUrl, venue?.autoPlay));
```

Comment:
This legacy function starts with a capital and has a boolean trap. @ermiaSparkle how much would it take to do a light change of its interface? Something like

```ts
export interface ConvertToEmbeddableUrlOptions {
  url?: string;
  autoplay?: boolean;
}

// @debt Replace the naive logic with use of new URL(url) which provides standard parsing before doing checks
export const convertToEmbeddableUrl: (
  options: ConvertToEmbeddableUrlOptions
) => string = ({ url, autoplay = true }) => {
```

and replace in the places it is invoked from?

Reply:
If it takes too much time, then at least 2 @debt lines: the one from the code example above and one to update the interface of it

Reply:
fixed in 418cdcb4763615a45870bc6b1dfd26c2ec198a84